css/open-licence.css

```diff
@@ -0,0 +1,189 @@
+@charset "utf-8";
```

Comment:
Hi @AshTNA - would you be OK to run auto format against this file (the indentation is inconsistent). Thanks 👍🏽